controllers/transactionsController.js

```diff
@@ -0,0 +1,29 @@
+const transactionsModel = require('../models/transaction')
+const { getUserId } = require('../utils/users')
+
+/**
+ * Collects all transactions and sends only required data for each userId
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const fetchLatest = async (req, res) => {
+  try {
+    const userId = await getUserId(req.params.username)
+    const noOfRecords = req.params.noOfRecords
```

Comment:
(read the comment on routes files first)
Once we have it as query param, since query params are optional, we should have a default value for number of transactions. Something like 10?

Reply:
Default value set to 10 if query param noOfRecords is not available in url